handles.h

```diff
@@ -5,6 +5,7 @@ namespace handles {
         TYPE_UNUSED,
 		TYPE_FILE,
 		TYPE_MAPPED,
+        TYPE_PROCESS
```

Comment:
Did indentation get messed up here?

Reply:
weird! It looks fine in my editor, but is showing up on main as well (https://github.com/decompals/wibo/blob/main/handles.h)

Reply:
Wibo code uses tabs instead of spaces. We need to add an .editor-config or similar.